modules/nf-core/gprofiler2/gost/meta.yml

```diff
@@ -0,0 +1,144 @@
+name: "gprofiler2_gost"
+description: runs a functional enrichment analysis with gprofiler2
+keywords:
+  - gene set analysis
+  - enrichment
+  - gprofiler2
+  - gost
+  - gene set
+tools:
+  - "gprofiler2":
+      description: "An R interface corresponding to the 2019 update of g:Profiler web tool."
+      homepage: "https://gl.cs.ut.ee/biit/r-gprofiler2"
```

Comment:
probably have this as the homepage instead? https://biit.cs.ut.ee/gprofiler/page/r
can still use the other link as the dev url below I think